app/components/form/instance-table.js

```diff
@@ -0,0 +1,61 @@
+import { action } from '@ember/object';
+import Component from '@glimmer/component';
+import { tracked } from '@glimmer/tracking';
+
+import { inject as service } from '@ember/service';
+
+export default class InstanceTableComponent extends Component {
+  @service store;
+
+  @tracked formInfo = null;
```

Comment:
I'd rather not have null in a dynamically typed language, but I guess it's inevitable

Reply:
a way around this would be to add a placeholder value while loading but i think this doesn't add too much (and would complicate the initialized getter)